eureka/templates/wagtailadmin/login.html

```diff
@@ -23,7 +23,7 @@
         {% if CAS_BASE %}
         <form method="get" action="{{ CAS_BASE }}cas/login">
             {% csrf_token %}
-            <input type="hidden" name="destination" value="https://{{ request.get_host }}/accounts/caslogin/?next={{ request.get_full_path|urlencode }}" />
```

Comment:
We've been leaving `next` in there as `?next=/`. I'm going to merge and see how this works until we get a broader fix in place.

Reply:
you're right, it needs `?next/` or it breaks, I'll make another PR